src/HID-APIs/D3MouseAPI.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <Arduino.h>
+#include "HID-Settings.h"
+
+typedef union {
+    uint8_t whole8[0];
+    uint16_t whole16[0];
+    uint32_t whole32[0];
+
+    struct
```

Comment:
Add packed attribute so the compiler does not add pad bytes between elements.

`struct ATTRIBUTE_PACKED`

Reply:
```suggestion
    struct ATTRIBUTE_PACKED
```